claripy/backends/backend_z3.py

```diff
@@ -156,6 +156,20 @@ def __init__(self, reuse_z3_solver=None, ast_cache_size=10000):
         self._op_raw['__xor__'] = self._op_xor
         self._op_raw['__and__'] = self._op_and
 
+        # String operations
+        self._op_raw['StrConcat'] = self._op_raw_StrConcat
+        self._op_raw['StrSubstr'] = self._op_raw_StrSubstr
+        self._op_raw['StrExtract'] = self._op_raw_StrExtract
+        self._op_raw['StrLen'] = self._op_raw_StrLen
+        self._op_raw['StrReplace'] = self._op_raw_StrReplace
+        self._op_raw['StrContains'] = self._op_raw_StrContains
+        self._op_raw['StrPrefixOf'] = self._op_raw_StrPrefixOf
+        self._op_raw['StrSuffixOf'] = self._op_raw_StrSuffixOf
+        self._op_raw['StrIndexOf'] = self._op_raw_StrIndexOf
+        self._op_raw['StrToInt'] = self._op_raw_StrToInt
+        self._op_raw['IntToStr'] = self._op_raw_IntToStr     
+        self._op_raw['UnitStr'] = self._op_raw_UnitStr 
+
```

Comment:
Is this necessary? It looks like there's code up above to loop over the class's methods with this naming scheme and automatically set them into this dictionary.

Reply:
Correct, I'll delete those in my commit tomorrow